src/components/NodesList/index.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import getNodesList from '../../redux/actions/nodeClusterActions';
+// import tellAge from '../../helpers/ageUtility';
+import './NodesList.css';
+import NavBar from '../NavBar';
+import Status from '../Status';
+import SpinnerComponents from '../SpinnerComponent';
+import InformationBar from '../InformationBar';
+import SideNav from '../SideNav';
+
+
+class NodesList extends Component {
+  componentDidMount() {
+    const { getNodesList } = this.props;
+    const { match: { params } } = this.props;
+    getNodesList(params.clusterID);
+  }
+
+  getAge(utcDate) {
+    const creationTimestamp = new Date(utcDate).getTime();
+    return creationTimestamp;
+  }
+
+  getRoles(node) {
+    if (typeof (node.spec.taints) !== 'undefined') {
+      const str = String(node.spec.taints[0].key);
+      // get strings after /
+      const role = str.substr(str.indexOf('/') + 1);
+      return role;
+    }
+    return '<none>';
+  }
+
+  nodeStatus(conditions) {
+    let status = '';
+    conditions.map((condition) => {
+      if (condition.type === 'Ready') {
+        status = condition.status;
+      }
+      return null;
+    });
+    if (status === 'True') {
+      return true;
+    }
+    return false;
+  }
+
+
+  render() {
+    const { nodes, isFetched, isRetrieving } = this.props;
+    const { clusterName } = this.props;
```

Comment:
This line is not working and not returning the required value, I have a fix for this in my PR.